drivers/hmis/app/graphql/mutations/publish_form.rb

```diff
@@ -0,0 +1,38 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Mutations
+  class PublishForm < CleanBaseMutation
+    argument :id, ID, required: true
+
+    field :form_definition, Types::Forms::FormDefinition, null: true
+
+    def resolve(id:)
+      access_denied! unless current_user.can_manage_forms?
+
+      definition = Hmis::Form::Definition.find_by(id: id)
+      raise 'not found' unless definition
+      raise 'only draft forms can be published' unless definition.status == Hmis::Form::Definition::DRAFT
+
+      previous = Hmis::Form::Definition.where(
```

Comment:
There can only be one published record for a given identifier, so this can be a `find_by`. That will make it so `previous` is a record instead of a scope. It might be clearer to rename to `previous_published_form` to clarify that.